### PR TITLE
OSDOCS-1837: Release note for Ingress TLS 1.3 support

### PR DESCRIPTION
**Pull request by skrthomas**
https://issues.redhat.com/browse/OSDOCS-1837

For 4.9

Preview: https://deploy-preview-36649--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-networking

@miheer ready for review.

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -147,6 +147,12 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-nw-tls-profile"]
+==== Support added for TLS 1.3 and the Modern Profile
```
Comment (miheer) [inline release_notes/ocp-4-9-release-notes.adoc:151]: Support added for TLSv1.3 and the Modern Profile

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -147,6 +147,12 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
...
+
+This release adds Ingress Controller support for TLS 1.3 and the `Modern` profile.
```
Comment (miheer) [inline release_notes/ocp-4-9-release-notes.adoc:153]: This release adds Ingress Controller support for using TLSv1.3 and the `Modern` profile in haproxy.

**Review by miheer — CHANGES_REQUESTED** [review]

**Comment by miheer** [comment]
@skrthomas 

In the product documentation, please delete statements that TLSv1.3 is not supported.

We also want to release notes in OCP 4.6, 4.7, 4.8 as follows -

In OCP 4.6, 4.7 and 4.8, 

The “Modern” profile will continue to be unsupported on OpenShift 4.6, 4.7, and 4.8; selecting it will continue to have the effect of enabling the “Intermediate” profile.
Specifying TLSv1.3 cipher suites will continue to have no effect; OpenShift router will enable Red Hat distributed OpenSSL’s default set of TLSv1.3 cipher suites which are  TLS_AES_128_CCM_SHA256,  TLS_CHACHA20_POLY1305_SHA256, TLS_AES_256_GCM_SHA384, TLS_AES_128_GCM_SHA256

**Review by jeana-redhat — COMMENTED** [review]
Just a couple small formatting things and a URL fix.

**Comment by skrthomas** [comment]
#33497 (RN Tracker)

**Pull request merged**